src/style.css

```diff
@@ -1,3 +1,72 @@
-body {
+/* General reset */
+* {
   margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+
+  --color-blue: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+header__logo-link {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.menu {
```

Comment:
every block should be in sparate file, please create folder blocks and add there files `header.css` and `menu.css` with styles for those blocks, then import them to `style.css`